docs/user-guide/configuration.md

```diff
@@ -11,12 +11,8 @@ weight: 120
 toc: true
 ---
 
-The `terraform-docs` configuration is a yaml file. This is a convenient way to
-share the configuation amongst teammates, CI, or other toolings. To do so you
-can use `-c` or `--config` flag which accepts name of the config file.
-
-Default name of this file is `.terraform-docs.yml`, and it will get picked up
-(if existed) without needing to explicitly passing with config flag.
```

Comment:
This is an important line that got dropped as part of this change.

Reply:
I think this might be better

```suggestion
The default name of this file is `.terraform-docs.yml`, and it will get picked up
(if existed) without needing to explicitly passing with config flag.
```